lesson_02/ht_template/job2/dal/save_avro.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Dict, Any
+
+import fastavro
+from fastavro import writer, parse_schema
+import shutil
+import os
+
+
+def save_data_to_avro(data: List[Dict[str, Any]], stg_dir: str, date: str) -> None:
```

Comment:
You use a generic name for this function (save_data_to_avro) and have a hardcoded sales data schema here. I would go either 2 ways:
1. Leave this function and rename it to `save_sales_to_avro`
2. Remove the sales schema and use the schema as an input parameter. Use the actual avro schema in BLL.

The way 2 is more prefered